airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -958,6 +962,19 @@ def execute(self, context: 'Context'):
                 context=context, task_instance=self, url=DATAPROC_JOB_LOG_LINK, resource=job_id
             )
 
+            if self.deferrable:
+                self.defer(
+                    trigger=DataprocBaseTrigger(
+                        job_id=self.job_id,
+                        project_id=self.project_id,
+                        region=self.region,
+                        delegate_to=self.delegate_to,
+                        gcp_conn_id=self.gcp_conn_id,
+                        impersonation_chain=self.impersonation_chain,
+                        pooling_period_seconds=10,
```

Comment:
can we not hardcode this value and use it as a param taken from the user as `polling_interval`